tests/examples/test_examples.py

```diff
@@ -182,6 +182,7 @@ def _assert_snapshot(example, url, example_type):
     success = result['success']
     timeout = result['timeout']
     errors = result['errors']
+    resources = result['resources']
 
     no_errors = len(errors) == 0
     no_resources = len(resources) == 0
```

Comment:
flake8 was complaining `resources` was undefined. I assume adding the line above was the intended solution.